src/tz.cpp

```diff
@@ -2658,9 +2668,8 @@ static
 std::vector<leap_second>
 find_read_and_leap_seconds()
 {
-    std::ifstream in(get_tz_dir() + std::string(1, folder_delimiter) + "leapseconds",
-                     std::ios_base::binary);
-    if (in)
+    auto in = get_stream("leapseconds");// (get_tz_dir() + std::string(1, folder_delimiter) + "leapseconds",std::ios_base::binary);
+    if (true)
```

Comment:
Can in be null? Better to just remove this line if so and you want to go down the codepath?

Reply:
It can't be null, we'd have already errored by this point, but the syntax is valid, scratch one line edit!